app/templates/README.mdown

```diff
@@ -71,26 +71,26 @@ Next, push all source files to the appropriate server. The following assumes it
 - sudo ln -s /space/projects/slush-app.live/etc/init.d/node-express-service slush-app
 - sudo chkconfig --add slush-app
 - sudo chkconfig --levels 2345 slush-app on
-  
+
 Next to start it, use the following commands (from any directory):
-  
+
 - sudo service slush-app start
-  
+
 These services will also print usage without param, but they support info, restart, start, status, and stop. The info param is very useful to check the settings.
 
 # Data
 
 ## Sample Data
 
-The application comes with 25 JSON documents generated by json-generator.com.
+The application comes with 1300 JSON documents generated by json-generator.com.
 You can load them with
 (MLCP)[https://docs.marklogic.com/guide/ingestion/content-pump] like this:
 
-    ./ml local mlcp import -input_file_path sample-data -document_type json
+    ./ml local mlcp import -input_file_path sample-data -document_type json -input_compressed true
```

Comment:
While in here, can you make some more changes to this line?

- You don't need the `-document_type` param, MarkLogic will auto-detect from extension.
- You can save yourself typing `true`, just `-input_compressed` is enough.
- Could you please add `-output_collections data`, both a good practice, and useful for changing the additional-query to a better default
- Last but not least, we should apply doc-permissions to allow the app-user that gets generated automatically access as well: `output_permissions <app-name>-user:read,<app-name>-user:update`

Oooh, we can automate the <app-name> replacement. Put the above options in an `import-sample-data.options` file, and use @ml.app-name inside there! :)